tests/drawer/test_tape_text.py

```diff
@@ -313,7 +361,9 @@ def test_add_op(self, op, out):
     )
     def test_add_mid_measure_op(self, op, layer_str, bit_map, out):
         """Test adding the first MidMeasureMP to array of strings"""
-        assert out == _add_op(op, layer_str, _Config(wire_map=default_wire_map, bit_map=bit_map))
+        print(out)
+        config = _Config(wire_map=default_wire_map, bit_map=bit_map, num_op_layers=4, cur_layer=0)
```

Comment:
Much nicer to read. If it's not too much hassle, might be nice to reformat other occurrences as well :sweat_smile: